data/strings/strings.txt

```diff
@@ -30845,3 +30845,233 @@
+    es = Listas eliminadas recientemente
+    et = Hiljuti kustutatud nimekirjad
+    eu = Duela gutxi ezabatutako zerrendak
+    fa = لیست های اخیرا حذف شده
+    fi = Äskettäin poistetut luettelot
+    fr = Listes récemment supprimées
+    he = רשימות שנמחקו לאחרונה
+    hi = हाल ही में हटाई गई सूचियाँ
+    hu = Nemrég törölt listák
+    id = Daftar yang Baru Saja Dihapus
+    it = Elenchi eliminati di recente
+    ja = 最近削除されたリスト
+    ko = 최근 삭제한 목록
+    lt = Neseniai ištrinti sąrašai
+    mr = अलीकडे हटवलेल्या याद्या
+    nb = Nylig slettede lister
+    nl = Onlangs verwijderde lijsten
+    pl = Ostatnio usunięte listy
+    pt = Listas eliminadas recentemente
+    pt-BR = Listas excluídas recentemente
+    ro = Liste șterse recent
+    ru = Недавно удаленные списки
+    sk = Nedávno odstránené zoznamy
+    sv = Nyligen borttagna listor
+    sw = Orodha Zilizofutwa Hivi Karibuni
+    th = รายการที่ถูกลบล่าสุด
+    tr = Son Silinen Listeler
+    uk = Нещодавно видалені списки
+    vi = Danh sách đã xóa gần đây
+    zh-Hans = 最近删除的列表
+    zh-Hant = 最近刪除的列表
+
+  [clear]
+    comment = Title for the "Clear" button on the Recently Deleted Lists screen.
+    tags = ios
+    en = Clear
+    af = Vee uit
+    ar = مسح
+    az = Sil
+    be = Ачысціць
+    bg = Изчистване
+    ca = Netejar
+    cs = Vymazat
+    da = Ryd
+    de = Löschen
+    el = Εκκαθάριση
+    es = Limpiar
+    et = Kustuta
+    eu = Garbitu
+    fa = پاک کردن
+    fi = Tyhjennä
+    fr = Effacer
+    he = נקה
+    hi = साफ करें
+    hu = Törlés
+    id = Hapus
+    it = Cancella
+    ja = クリア
+    ko = 지우기
+    lt = Išvalyti
+    mr = साफ करा
+    nb = Fjern
+    nl = Wissen
+    pl = Wyczyść
+    pt = Limpar
+    pt-BR = Limpar
+    ro = Șterge
+    ru = Очистить
+    sk = Vymazať
+    sv = Rensa
+    sw = Futa
+    th = ล้าง
+    tr = Temizle
+    uk = Очистити
+    vi = Xóa
+    zh-Hans = 清除
+    zh-Hant = 清除
+
+  [delete_all]
+    comment = Title for the "Delete All" button on the Recently Deleted Lists screen.
+    tags = ios
+    en = Delete All
+    af = Vee alles uit
+    ar = حذف الكل
+    az = Hamısını Sil
+    be = Выдаліць усе
+    bg = Изтриване на всички
+    ca = Eliminar tots
+    cs = Smazat vše
+    da = Slet alle
+    de = Alle löschen
+    el = Διαγραφή όλων
+    es = Borrar todo
+    et = Kustuta kõik
+    eu = Ezabatu guztiak
+    fa = حذف همه
+    fi = Poista kaikki
+    fr = Supprimer tout
+    he = מחק הכל
+    hi = सभी हटा दो
+    hu = Mindent törölni
+    id = Hapus Semua
+    it = Cancella tutto
+    ja = すべて削除する
+    ko = 모두 삭제
+    lt = Ištrinti viską
+    mr = सर्व हटवा
+    nb = Slett alle
+    nl = Alles verwijderen
+    pl = Usuń wszystko
+    pt = Eliminar tudo
+    pt-BR = Excluir tudo
+    ro = Ștergeți toate
+    ru = Удалить все
+    sk = Odstrániť všetko
+    sv = Radera alla
+    sw = Futa Zote
+    th = ลบทั้งหมด
+    tr = Tümünü Sil
+    uk = Видалити все
+    vi = Xóa hết
+    zh-Hans = 全部删除
+    zh-Hant = 刪除所有
+
+  [recover]
```

Comment:
Apple uses Recover word in Files app. We use Restore on accidentally deleted bookmarks on the map. Should the existing Restore translation for bookmarks on the map be replaced with Recover? It can be done later in a separate PR.